packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.0
+
+* Adds a new interface for initializing map renderer.
```

Comment:
This should not be part of the platform interface, only the Android implementation. See https://github.com/flutter/plugins/blob/main/packages/google_maps_flutter/google_maps_flutter_android/README.md#display-mode for the structure this should follow.